nncf/common/quantization/initialization/range.py

```diff
@@ -160,6 +160,28 @@ def __init__(self, is_weights: bool, mode: QuantizationMode, per_channel: bool):
         self._mode = mode
         self._per_channel = per_channel
 
+    @property
+    def is_weights(self) -> bool:
+        """
+        Returns boolean that defines tensor type.
+        True for Weights, False for Activations.
+        """
+        return self._is_weights
+
+    @property
+    def mode(self) -> QuantizationMode:
+        """
+        Returns quantization mode: symmetric or asymmetric.
+        """
+        return self._mode
+
+    @property
+    def per_channel(self) -> bool:
+        """
+        Returns quantization granularity.
+        """
+        return self._per_channel
```

Comment:
Since `_per_channel` contains only a `boolean` value, why not call it `_is_per_channel`? To align it with `_is_weights`.

Reply:
Done